bin/check-http-json.rb

```diff
@@ -74,6 +74,11 @@ class CheckJson < Sensu::Plugin::Check::CLI
   option :dump_json, short: '-d', long: '--dump-json', boolean: true, default: false
   option :pretty, long: '--pretty', boolean: true, default: false
 
+  option :response_code,
+         long: '--response-code REGEX',
+         description: 'Check for a specific response code REGEX',
+         default: '^2'
```

Comment:
I wonder if maybe `^2([0-9]{2})$` might be a better default regex as its more explicit.